cmd/validate_image.go

```diff
@@ -113,7 +108,13 @@ Use an EnterpriseContractPolicy resource from a different namespace:
   ec validate image --image registry/name:tag --policy my-namespace/my-policy
 
 Use an inline EnterpriseContractPolicy spec
-  ec validate image --image registry/name:tag --policy '{"publicKey": "<path/to/public/key>"}'`,
+  ec validate image --image registry/name:tag --policy '{"publicKey": "<path/to/public/key>"}'
+
+Write output in JSON format to a file
+  ec validate image --image registry/name:tag --output json=<path>
+
+Write output in YAML format to stdout and in HACBS format to a file
+  ec validate image --image registry/name:tag --output yaml --output hacbs=<path>`,
```

Comment:
I like it.

Reply:
I was initially thinking maybe the json vs yaml would orthogonal to the structure, e.g. what if I want the hacbs shaped output in yaml format? But now I feel like this is fine. It's easy enough to expand the output format, `hacbs-yaml`, `hacbs-xml` etc if we feel the need.

Reply:
Yeah. The reason I added yaml is that it's much easier on the eyes and makes it good for throwing in the logs. I could've gone with something like `jsonpretty`. yaml just seemed like a neater solution here.